[2025-05-21T11:56:30.956+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: telegram_summary_channel_26769.summarize_daily_posts manual__2025-05-21T11:56:27.936664+00:00 [queued]>
[2025-05-21T11:56:30.962+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: telegram_summary_channel_26769.summarize_daily_posts manual__2025-05-21T11:56:27.936664+00:00 [queued]>
[2025-05-21T11:56:30.962+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-21T11:56:30.970+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): summarize_daily_posts> on 2025-05-21 11:56:27.936664+00:00
[2025-05-21T11:56:30.974+0000] {standard_task_runner.py:60} INFO - Started process 264 to run task
[2025-05-21T11:56:30.979+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'telegram_summary_channel_26769', 'summarize_daily_posts', 'manual__2025-05-21T11:56:27.936664+00:00', '--job-id', '249', '--raw', '--subdir', 'DAGS_FOLDER/dag_generator.py', '--cfg-path', '/tmp/tmp_x3p4pdp']
[2025-05-21T11:56:30.981+0000] {standard_task_runner.py:88} INFO - Job 249: Subtask summarize_daily_posts
[2025-05-21T11:56:31.026+0000] {task_command.py:423} INFO - Running <TaskInstance: telegram_summary_channel_26769.summarize_daily_posts manual__2025-05-21T11:56:27.936664+00:00 [running]> on host 7410cbfeba7d
[2025-05-21T11:56:31.105+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='telegram_summary_channel_26769' AIRFLOW_CTX_TASK_ID='summarize_daily_posts' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T11:56:27.936664+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T11:56:27.936664+00:00'
[2025-05-21T11:56:31.158+0000] {docker.py:359} INFO - Starting docker container from image telegram_summarizer_task:latest
[2025-05-21T11:56:31.598+0000] {docker.py:429} INFO - [2025-05-21 11:56:31] [INFO] [Summarizer] - Запуск основного процесса суммризатора (локальный или прямой вызов)...
[2025-05-21 11:56:31] [INFO] [Summarizer] - --- Запуск скрипта суммризации (Airflow Task) ---
[2025-05-21 11:56:31] [INFO] [Summarizer] - Попытка разбора XCom из переменной окружения 'XCOM_DATA_JSON'...
[2025-05-21 11:56:31] [INFO] [Summarizer] - Прочитаны данные XCom: channel_id=26769, target_date_str=2025-05-21, parser_status=failure
[2025-05-21 11:56:31] [WARNING] [Summarizer] - Статус предыдущей задачи парсинга: 'failure'. Саммаризация может быть неполной или основана на ошибочных данных.
[2025-05-21T11:56:31.598+0000] {docker.py:429} INFO - [2025-05-21 11:56:31] [INFO] [Summarizer] - Парсер не нашел или не сохранил сообщений. Суммаризация не требуется.
[2025-05-21 11:56:31] [WARNING] [Summarizer] - Скрипт суммризации завершается через sys.exit() с кодом: 0
[2025-05-21 11:56:31] [INFO] [Summarizer] - --- Скрипт суммризации завершен с кодом выхода: 0 ---
[2025-05-21 11:56:31] [INFO] [Summarizer] - Суммризатор: Перехвачен SystemExit с кодом 0. Завершение.
[2025-05-21 11:56:31] [INFO] [Summarizer] - Скрипт суммризатора окончательно завершается с кодом 0 (из __main__ блока).
[2025-05-21T11:56:31.967+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=telegram_summary_channel_26769, task_id=summarize_daily_posts, execution_date=20250521T115627, start_date=20250521T115630, end_date=20250521T115631
[2025-05-21T11:56:31.996+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-21T11:56:32.017+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
